src/pkg/quota/quota.go

```diff
@@ -6,9 +6,18 @@ import (
 	"net/url"
 	"strings"
 
-	defangv1 "github.com/DefangLabs/defang/src/protos/io/defang/v1"
+	compose "github.com/compose-spec/compose-go/v2/types"
 )
 
+const Mode_INGRESS = "ingress"
```

Comment:
need a better place for these

Reply:
Yes, quota.Mode_INGRESS is quite strange, but let's put a TODO here for now.

Reply:
Moved to separate file